ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -237,10 +249,19 @@ export class PrepareMigrationComponent implements OnInit {
             if (res.Message.startsWith('Schema migration complete')) {
               localStorage.setItem(MigrationDetails.SchemaMigrationProgress, "100")
               this.schemaMigrationProgress = parseInt(localStorage.getItem(MigrationDetails.SchemaMigrationProgress) as string)
-              if (this.selectedMigrationMode == MigrationModes.schemaOnly || this.selectedMigrationType == MigrationTypes.lowDowntimeMigration) {
+              if (this.selectedMigrationMode == MigrationModes.schemaOnly) {
                 this.markMigrationComplete()
+              } else if (this.selectedMigrationType == MigrationTypes.lowDowntimeMigration && !displayStreamingMsg) {
+                this.markSchemaMigrationComplete()
+                this.hasDataMigrationStarted = false
+                localStorage.setItem(MigrationDetails.HasDataMigrationStarted, this.hasDataMigrationStarted.toString())
+                this.snack.openSnackBarWithoutTimeout('Setting up dataflow and datastream jobs', 'Close')
+                displayStreamingMsg = true
               }
             }
+            else if (res.Message.startsWith('Data migration complete')) {
```

Comment:
Performing logic like this on the basis of text is a bad idea. It is always better to pass a status field and text separately and use the status field for these checks, and text to just be shown.

Reply:
okay will add a status field in the progress object